src/marks/tree.d.ts

```diff
@@ -6,10 +6,47 @@ import type {TextOptions} from "./text.js";
 
 // TODO tree channels, e.g., "node:name" | "node:path" | "node:internal"?
 export interface TreeOptions extends DotOptions, LinkOptions, TextOptions, TreeTransformOptions {
+  /**
+   * Whether to represent the node with a dot; defaults to true unless a
+   * **marker** is specified.
+   */
   dot?: boolean;
+  /**
+   * The **stroke** color for the text mark to improve the legibility of labels
+   * atop other marks by creating a halo effect; defaults to *white*.
+   */
   textStroke?: MarkOptions["stroke"];
 }
 
+/**
+ * Transforms a tabular dataset into a hierarchy according to the given **path**
+ * input channel, which typically contains slash-separated strings; then
+ * executes a tree layout algorithm, by default Reingold–Tilford’s “tidy”
+ * algorithm, to compute *x* and *y* output channels; these channels can then be
+ * fed to other marks to construct a node-link diagram.
```

Comment:
This sounds like it’s describing the tree transform, whereas here we should be describing the tree compound mark, which means this function is doing the “feeding” (to a link, dot, and text mark).